redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -126,6 +138,11 @@ object Input {
       Chunk(encodeString("LIMIT"), encodeString(data.offset.toString), encodeString(data.count.toString))
   }
 
+  final case class ListInput[-A](input: Input[A]) extends Input[List[A]] {
+    override private[redis] def encode(data: List[A]): Chunk[RespValue.BulkString] =
+      data.foldLeft(Chunk.empty: Chunk[RespValue.BulkString])((acc, a) => acc ++ input.encode(a))
```

Comment:
compiler was very unhappy about `Chunk(data.map(input.encode))` ... may come back to figure out how to make this more clear.